packages/expo-linear-gradient/ios/LinearGradientLayer.swift

```diff
@@ -5,7 +5,7 @@ import CoreGraphics
 
 var defaultStartPoint = CGPoint(x: 0.5, y: 0.0)
 var defaultEndPoint = CGPoint(x: 0.5, y: 1.0)
-var defaultLocations: [CGFloat] = [0.0, 1.0]
+var defaultLocations: [CGFloat] = []
```

Comment:
I can see that our docs are explicitly documenting this behaviour https://docs.expo.dev/versions/latest/sdk/linear-gradient/#lineargradientprops 🤔

Reply:
I dug a bit deeper, and also removed the default value from the type definition that I overlooked initially.

Reply:
Yeah, I've got nothing against removing it in favour of something different, e.g. empty array.
And what about other platforms? Web and Android? Is the new default value propagated there as well? 🤔 
We don't want to introduce any discrepancies between something that is consistent right now.